axon-framework/modules.md

```diff
@@ -12,22 +12,22 @@ Axon 'Main Modules' are the modules that have been thoroughly tested and are rob
 
 > **Note**
 >
-> The [Axon Spring Boot Starter](maven-dependencies.md#axon-spring-boot-starter) module is the quickest start in to an Axon project as it will retrieve all the required modules/dependencies transitively. Alternatively, you can manually select individual modules for a customized configuration.
+> The [Axon Spring Boot Starter]() module is the quickest start in to an Axon project as it will retrieve all the required modules/dependencies transitively. Alternatively, you can manually select individual modules for a customized configuration.
 
 | Module | Artifact Id | Group Id | Maven Central |
 | :--- | :--- | :--- | :---: |
-| [Axon Messaging](maven-dependencies.md#axon-messaging) | axon-messaging | org.axonframework | [available](https://search.maven.org/search?q=a:axon-messaging) |
-| [Axon Modeling](maven-dependencies.md#axon-modeling) | axon-modelling | org.axonframework | [available](https://search.maven.org/search?q=a:axon-modelling) |
-| [Axon Event Sourcing](maven-dependencies.md#axon-event-sourcing) | axon-eventsourcing | org.axonframework | [available](https://search.maven.org/search?q=a:axon-eventsourcing) |
-| [Axon Configuration](maven-dependencies.md#axon-configuration) | axon-configuration | org.axonframework | [available](https://search.maven.org/search?q=a:axon-configuration) |
-| [Axon Test](maven-dependencies.md#axon-test) | axon-test | org.axonframework | [available](https://search.maven.org/search?q=a:axon-test) |
-| [Axon Server Connector](maven-dependencies.md#axon-server-connector) | axon-server-connector | org.axonframework | [available](https://search.maven.org/search?q=a:axon-server-connector) |
-| [Axon Spring](maven-dependencies.md#axon-spring) | axon-spring | org.axonframework | [available](https://search.maven.org/search?q=a:axon-spring) |
-| [Axon Spring Boot Starter](maven-dependencies.md#axon-spring-boot-starter) | axon-spring-boot-starter | org.axonframework | [available](https://search.maven.org/search?q=a:axon-spring-boot-starter) |
-| [Axon Disruptor](maven-dependencies.md#axon-disruptor) | axon-disruptor | org.axonframework | [available](https://search.maven.org/search?q=a:axon-disruptor) |
-| [Axon Metrics](maven-dependencies.md#axon-metrics) | axon-metrics | org.axonframework | [available](https://search.maven.org/search?q=a:axon-metrics) |
-| [Axon Micrometer](maven-dependencies.md#axon-micrometer) | axon-micrometer | org.axonframework | [available](https://search.maven.org/search?q=a:axon-micrometer) |
-| [Axon Legacy](maven-dependencies.md#axon-legacy) | axon-legacy | org.axonframework | [available](https://search.maven.org/search?q=a:axon-legacy) |
+| [Axon Messaging]() | axon-messaging | org.axonframework | [available](https://search.maven.org/search?q=a:axon-messaging) |
```

Comment:
Why have the links to subsections in this document been removed actually?

Reply:
Resolved